storage/reads/aggregate_resultset_test.go

```diff
@@ -0,0 +1,168 @@
+package reads_test
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/influxdata/influxdb/v2/models"
+	"github.com/influxdata/influxdb/v2/storage/reads"
+	"github.com/influxdata/influxdb/v2/storage/reads/datatypes"
+	"github.com/influxdata/influxdb/v2/tsdb/cursors"
+)
+
+func TestNewWindowAggregateResultSet_Tags(t *testing.T) {
+
+	newCursor := sliceSeriesCursor{
+		rows: newSeriesRows(
+			"clicks click=1 1",
+		)}
+
+	request := datatypes.ReadWindowAggregateRequest{}
+	resultSet, err := reads.NewWindowAggregateResultSet(context.Background(), &request, &newCursor)
+
+	if err != nil {
+		t.Errorf("Error creating WindowAggregateResultSet: %s", err)
+	}
+
+	// If .Next() was never called, seriesRow is nil and tags are empty.
+	expectedTags := "[]"
+	if resultSet.Tags().String() != expectedTags {
+		t.Errorf("Expected tags: %s got: %s", expectedTags, resultSet.Tags().String())
```

Comment:
Similar thing here for the message. A good preference is to try to create an error message that looks like this:

```
unexpected tags -want/+got:
    - wanted tags
    + got tags
```

A good template for this is:

```
if want, got := expectedTags, resultSet.Tags().String(); want != got {
    t.Errorf("unexpected tags -want/+got:\n\t- %s\n\t+ %s", want, got)
}
```

If you notice, I also assign the values we are comparing to `want` and `got` and then compare them. This helps in two ways. First, it makes the comparison and then subsequent printing easier. Second, it uses a standard name so it is easier to read and know what you are expecting so you can more easily keep track of the got and wanted.

For whatever reason, we use the word `want` instead of `expected` or `exp`.

`Errorf` is probably fine here since the remainder of the function can still run if this fails. The general rule of thumb is if the error is fatal and the unit test can't continue to use fatal, but try to use error if possible since you will have more problems printed out so fewer runs of the test.